eye_colors:
  black : [0,0,0]
  #tonos 1
  white : [200,200,1]
  red : [200,0,0]
  green : [0,200,0]
  blue : [0,0,200]
  yellow : [200,200,0]
  cian : [0,200,100]
  magenta : [200,0,100]

  #tonos 2
  white2 : [200,200,200]
  red2 : [200,0,0]
  green2 : [0,200,0]
  blue2 : [0,0,200]
  yellow2 : [200,200,0]
  cian2 : [0,200,200]
  magenta2 : [200,0,200]

  #tonos 3
  white3 : [230,230,230]
  red3 : [230,0,0]
  green3 : [0,230,0]
  blue3 : [0,0,230]
  yellow3 : [230,230,0]
  cian3 : [0,230,230]
  magenta3 : [230,0,230]
